variables.tf

```diff
@@ -144,4 +149,23 @@ variable "tables" {
   type        = list(string)
   description = "List of Storage Tables"
   default     = []
+}
+
+// CORS
+variable "cors" {
```

Comment:
Can you add some validation for the 'allowed_methods' please.
https://www.terraform.io/language/values/variables#custom-validation-rules